### PR TITLE
LOC CHECKIN 20200709 | dotnet-roslyn release/dev16.7-vs-deps  

### PR DESCRIPTION
**Pull request by cristianosuzuki77**

### CODE REVIEW

**Comment by wli3** [comment]
@JoeRobich there will be a Roslyn insertion later right?

**Comment by JoeRobich** [comment]
> @JoeRobich there will be a Roslyn insertion later right?

@wli3 We plan to take this and a few other changes for Preview 5 next week.

File: src/Analyzers/CSharp/Analyzers/xlf/CSharpAnalyzersResources.zh-Hans.xlf [hunk 1]
```diff
@@ -64,22 +64,22 @@
...
       <trans-unit id="Remove_suppression_operators">
         <source>Remove suppression operators</source>
-        <target state="new">Remove suppression operators</target>
+        <target state="translated">请删除禁止使用的运算符</target>
         <note />
       </trans-unit>
       <trans-unit id="Remove_unnecessary_suppression_operator">
         <source>Remove unnecessary suppression operator</source>
-        <target state="new">Remove unnecessary suppression operator</target>
+        <target state="translated">请删除不必要的禁止使用的运算符</target>
```
Comment (huoyaoyuan) [inline src/Analyzers/CSharp/Analyzers/xlf/CSharpAnalyzersResources.zh-Hans.xlf:82]:
```suggestion
        <target state="translated">请删除不必要的忽略运算符</target>
```

File: src/Analyzers/CSharp/Analyzers/xlf/CSharpAnalyzersResources.zh-Hans.xlf [hunk 2]
```diff
@@ -64,22 +64,22 @@
...
       <trans-unit id="Remove_operator_preserves_semantics">
         <source>Remove operator (preserves semantics)</source>
-        <target state="new">Remove operator (preserves semantics)</target>
+        <target state="translated">请删除运算符(保留语义)</target>
         <note />
       </trans-unit>
       <trans-unit id="Remove_suppression_operators">
         <source>Remove suppression operators</source>
-        <target state="new">Remove suppression operators</target>
+        <target state="translated">请删除禁止使用的运算符</target>
```
Comment (huoyaoyuan) [inline src/Analyzers/CSharp/Analyzers/xlf/CSharpAnalyzersResources.zh-Hans.xlf:77]:
```suggestion
        <target state="translated">请删除忽略运算符</target>
```

File: src/Compilers/CSharp/Portable/xlf/CSharpResources.zh-Hans.xlf [hunk 3]
```diff
@@ -94,22 +94,22 @@
...
       <trans-unit id="ERR_BadFuncPointerParamModifier">
         <source>'{0}' cannot be used as a modifier on a function pointer parameter.</source>
-        <target state="new">'{0}' cannot be used as a modifier on a function pointer parameter.</target>
+        <target state="translated">“{0}”不能作为函数指针参数上的修饰符使用。</target>
         <note />
       </trans-unit>
       <trans-unit id="ERR_BadInheritanceFromRecord">
         <source>Only records may inherit from records.</source>
-        <target state="new">Only records may inherit from records.</target>
+        <target state="translated">只有记录可以从记录中继承。</target>
```
Comment (huoyaoyuan) [inline src/Compilers/CSharp/Portable/xlf/CSharpResources.zh-Hans.xlf:107]:
```suggestion
        <target state="translated">只有记录可以从记录继承。</target>
```

File: src/Compilers/CSharp/Portable/xlf/CSharpResources.zh-Hans.xlf [hunk 4]
```diff
@@ -199,7 +199,7 @@
       </trans-unit>
       <trans-unit id="ERR_CopyConstructorMustInvokeBaseCopyConstructor">
         <source>A copy constructor in a record must call a copy constructor of the base, or a parameterless object constructor if the record inherits from object.</source>
-        <target state="new">A copy constructor in a record must call a copy constructor of the base, or a parameterless object constructor if the record inherits from object.</target>
+        <target state="translated">如果记录继承自对象，则记录中的复制构造函数必须调用基对象的复制构造函数，或者调用无参数的对象构造函数。</target>
```
Comment (huoyaoyuan) [inline src/Compilers/CSharp/Portable/xlf/CSharpResources.zh-Hans.xlf:202]:
```suggestion
        <target state="translated">如果记录继承自 object ，则记录中的复制构造函数必须调用基对象的复制构造函数，或者调用无参数的对象构造函数。</target>
```

File: src/Compilers/CSharp/Portable/xlf/CSharpResources.zh-Hans.xlf [hunk 5]
```diff
@@ -234,12 +234,12 @@
       </trans-unit>
       <trans-unit id="ERR_DoesNotOverrideMethodFromObject">
         <source>'{0}' does not override the method from 'object'.</source>
-        <target state="new">'{0}' does not override the method from 'object'.</target>
+        <target state="translated">“{0}”不会通过 "object" 覆盖方法。</target>
```
Comment (huoyaoyuan) [inline src/Compilers/CSharp/Portable/xlf/CSharpResources.zh-Hans.xlf:237]:
```suggestion
        <target state="translated">“{0}”不会覆盖 "object" 的方法。</target>
```

**Comment by wli3** [comment]
@huoyaoyuan thank you for your suggestions. However due to the process, your advice will not reflect to the code in a short time.

@janbaltus could you incorporate the suggestion into your database?

@JoeRobich please go ahead and merge this change. A later loc pass will have the suggestions.

File: src/Compilers/CSharp/Portable/xlf/CSharpResources.zh-Hans.xlf (hunk 5 above)
Comment (janbaltus) [reply src/Compilers/CSharp/Portable/xlf/CSharpResources.zh-Hans.xlf:237]: we are working on review of these suggestions

**Comment by wli3** [comment]
@JoeRobich ping

**Comment by JoeRobich** [comment]
@wli3 Still waiting on QB approval for a Preview 5 insertion.

**Comment by cristianosuzuki77** [comment]
Suggestions were incorporated.

**Pull request merged**